pkg/formatted/k8s.go

```diff
@@ -24,10 +24,13 @@ var ConditionColor = map[string]color.Attribute{
 	"Failed":    color.FgHiRed,
 	"Succeeded": color.FgHiGreen,
 	"Running":   color.FgHiBlue,
-	"Cancelled": color.FgHiMagenta,
+	"Cancelled": color.FgHiWhite,
+	"Completed": color.FgHiMagenta,
+	"Pending":   color.FgHiYellow,
```

Comment:
Would it be better to do Yellow for Cancelled and White for Pending? Pending is very unlikely to be seen as `PodInitializing` status actually happens really quick. Maybe it makes more sense to use color to call attention to cancelation?

Reply:
I have no preference :D but your suggestions sounds good!